.github/workflows/coverage.yml

```diff
@@ -40,4 +40,4 @@ jobs:
           pytest --cov=networkx --runslow -n auto --doctest-modules --durations=20 --pyargs networkx
 
       - name: Upload to codecov
-        uses: codecov/codecov-action@v3
+        uses: codecov/codecov-action@v5
```

Comment:
With https://github.com/networkx/networkx/pull/8177/ we should be able to remove this action all together.